Formula/clamav.rb

```diff
@@ -44,6 +45,7 @@ class Clamav < Formula
   def install
     args = std_cmake_args + %W[
       -DAPP_CONFIG_DIRECTORY=#{etc}/clamav
+      -DDATABASE_DIRECTORY=#{var}/clamav
```

Comment:
Since it's in upstream's default config 

```suggestion
      -DDATABASE_DIRECTORY=#{var}/lib/clamav
```

Reply:
Yeah, upon further thought, that seems more "permanent" than `var/"cache"`, which befits the nature of AV signature DBs.